src/main/java/net/dv8tion/jda/api/interactions/commands/build/CommandData.java

```diff
@@ -37,189 +40,43 @@
  *
  * @see Commands
  */
-public interface CommandData extends SerializableData
+public interface CommandData
+        extends INamedCommandData, IScopedCommandData, IRestrictedCommandData, IAgeRestrictedCommandData,
+        SerializableData
 {
-    /**
-     * The maximum length the name of a command can be. ({@value})
-     */
-    int MAX_NAME_LENGTH = 32;
```

Comment:
Are these variables still accessible via `CommandData.MAX_NAME_LENGTH`? if not, this is technically a breaking change

Reply:
Yes, all fields can be accessed from subclasses